docs/community_resources/translations.md

```diff
@@ -5,6 +5,54 @@ If you'd like to see eZ Platform in your language, you can contribute to the tra
 [`ezplatform-i18n`](https://github.com/ezsystems/ezplatform-i18n) contains the XLIFF files providing translations.
 You can use an XLIFF editor of your choice to contribute strings in your language.
 
+## How to translate the interface using Crowdin
+
+If you wish to contribute to an existing translation of AdminUI or start a new one, the best way is to start with in-context translation (but you can also [translate directly on the Crowdin website](#translating-outside-in-context)).
+
+## Preparing to use in-context
+
+To start translating, you need an option to turn in-context translation on and off. To do this, set a browser cookie. There are several ways to do this, we will highlight a couple here.
+
+### Using bookmarks
+
+You can easily create a bookmark to toggle in-context on/off.
+
+Right-click your browser's bookmark bar, and create a new bookmark as shown in the screenshot below:
+
+![Bookmark for toggling in-context translation](img/toggle_incontext.png "Bookmark for toggling in-context translation")
+
+**Name**: Toggle in-context
+
+**URL**: `javascript:function hasInContextCookie(){ return document.cookie.match(/^(.*;)?\s*ez_in_context_translation\s*=\s*[^;]+(.*)?$/);}(function () { document.cookie = hasInContextCookie() ? document.cookie = &#39;ez_in_context_translation=;expires=Mon, 05 Jul 2000 00:00:00 GMT;path=/;&#39;: document.cookie=&#39;ez_in_context_translation=1;path=/;&#39;; location.reload()})()`
+
+Enter AdminUI. While on the page clicking this bookmark will turn on and off in-context translation for eZ Platform interface.
+
+### Using the debugging console
+
+Another way is to open the development console and run this command:
+
+``` javascript
+function hasInContextCookie(){ return document.cookie.match(/^(.*;)?\s*ez_in_context_translation\s*=\s*[^;]+(.*)?$/);}(function () { document.cookie = hasInContextCookie() ? document.cookie = 'ez_in_context_translation=;expires=Mon, 05 Jul 2000 00:00:00 GMT;path=/;': document.cookie='ez_in_context_translation=1;path=/;'; location.reload()})()
```

Comment:
Maybe use the shorter command from here: https://discuss.ezplatform.com/t/how-to-help-to-translate-ez-platform-v2/425